src/transformers/models/trocr/modeling_trocr.py

```diff
@@ -891,13 +891,46 @@ def forward(
         Example:
 
         ```python
-        >>> from transformers import VisionEncoderDecoderModel, TrOCRForCausalLM, ViTModel, TrOCRConfig, ViTConfig
-
+        >>> from transformers import (
+        ...     TrOCRProcessor,
+        ...     VisionEncoderDecoderModel,
+        ...     TrOCRForCausalLM,
+        ...     ViTModel,
+        ...     TrOCRConfig,
+        ...     ViTConfig,
+        ... )
```

Comment:
just a tiny thing, but would be really great if this could be in some order,  like

```python
        ...     TrOCRConfig,
        ...     TrOCRProcessor,
        ...     TrOCRForCausalLM,
        ...     ViTConfig,
        ...     ViTModel,
        ...     VisionEncoderDecoderModel,
```

Reply:
Done